### ISSUE TITLE
Compatibility with guard villagers?

### ISSUE THREAD

**Issue by SquareBall789** [ewewukek/mc-musketmod#52]
### Request information

Would be very nice
https://modrinth.com/mod/guard-villagers-(fabricquilt)

**Comment by Exokyhh** [comment]
Indeed! Same as pillagers and skeleton. A small chance of spawning with guns, or atleast being able to use the guns correctly. I tried, sometimes they shoot but they spin in circles randomly

**Comment by Helpful-Neko** [comment]
I would like to see this feature added too. I can imagine the scooped musket would be useless for them but all the other guns would be usable
